tests/lib/rules/indent.js

```diff
@@ -3424,6 +3424,133 @@ ruleTester.run("indent", rule, {
                 )
             `
         },
+        {
+            code: unIndent`
+                const foo = a.b(),
+                    longName =
+                        (memberFunction(
```

Comment:
I agree. When I first found this issue it was with a member function `foo.bar()` but after more testing it also happened with call expressions. I will update the test.